destination.go

```diff
@@ -43,6 +45,8 @@ type Destination struct {
 type Config struct {
 	// url to gRPC server
 	URL string `json:"url" validate:"required"`
+	// the bandwidth limit in bytes/second
+	RateLimit int `json:"rateLimit" default:"10000"`
```

Comment:
How is the rate limit disabled? By setting it to a high number or ?

Reply:
I created this before we talked about disabling it 👀 this was the default value, will check how to disable if possible